web/src/OrganizationEditPage.js

```diff
@@ -360,7 +360,7 @@ class OrganizationEditPage extends React.Component {
           </Col>
           <Col span={22} >
             <Select virtual={false} style={{width: "100%"}} value={this.state.organization.defaultApplication} onChange={(value => {this.updateOrganizationField("defaultApplication", value);})}
-              options={this.state.applications?.map((item) => Setting.getOption(item.name, item.name))
+              options={this.state.applications?.map((item) => Setting.getOption(item.isShared ? `${item.name}(shared)` : item.name, item.name))
```

Comment:
Define `Setting.getApplicationName(application)` to return your "XXX (Shared)" string.

"XXX (Shared)", not "XXX(shared)"